lib/universal-search.js

```diff
@@ -175,18 +175,19 @@ Search.prototype = {
     });
     app.urlbar.init();
 
-    app.highlightManager = new app.HighlightManager({
-      win: win,
-      events: app.events,
-      recommendation: app.recommendation
-    });
-    app.highlightManager.init();
-
     app.popup = new app.Popup({
       win: win,
       events: app.events
     });
     app.popup.init();
+
+    app.highlightManager = new app.HighlightManager({
+      win: win,
+      events: app.events,
+      recommendation: app.recommendation,
+      popup: app.popup
```

Comment:
adjusting initialization order so we can pass the popup object into the highlightManager